radicle-crdt/src/clock.rs

```diff
@@ -53,6 +62,36 @@ impl From<u64> for Lamport {
     }
 }
 
+impl From<Lamport> for u64 {
+    fn from(arg: Lamport) -> Self {
+        arg.get()
+    }
+}
+
+/// Error decoding an operation from an entry.
+#[derive(Error, Debug)]
+pub enum LamportError {
+    #[error("invalid Lamport Clock value")]
+    Invalid(),
```

Comment:
It might be useful to carry the parse error inside here and use the `#[source]` pragma

Reply:
How helpful would it be?  It feels redundant as Lamport Clocks don't have an underlying structure like SSH Keys or otherwise.

Reply:
It's useful to know what went wrong when trying to parse -- this time it's really for the developers of this code base. The `#[source]` will provide the error that occurred when parsing the `u64` -- which is arguably the underlying structure of the clock.